x-pack/plugins/timelines/public/components/t_grid/body/index.tsx

```diff
@@ -720,6 +728,7 @@ export const BodyComponent = React.memo<StatefulBodyProps>(
         columnHeaders,
         data,
         defaultCellActions,
+        visibleCellActions,
```

Comment:
instead of passing this down as a prop from all the places it's used, wouldn't it be safer to just hard code the 3 in this component?

Reply:
Yeah that would be fine! I wasn't sure what all plugins used this TGrid component and if future usages might want to customize `visibleCellActions`, so was trying to play it safe. If you're confident all future usages will want a constant max of 3 visible cell actions I'm good with hard-coding it.

Reply:
https://github.com/elastic/kibana/pull/129653/commits/fc934748cd11ec83daef3dd10c6cd1ac97a65609 @kqualters-elastic @andrew-goldstein. Here's the relevant line where `3` is hard-coded:

https://github.com/elastic/kibana/blob/fc934748cd11ec83daef3dd10c6cd1ac97a65609/x-pack/plugins/timelines/public/components/t_grid/body/index.tsx#L719